app/helpers/value_set_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ValueSetHelper
+  def direct_reference_code_hash(code_system_name, code_system_version, code)
+    'drc-' + Digest::SHA2.hexdigest("#{code_system_name} #{code['id']} #{code['name']} #{code_system_version}")
```

Comment:
Maybe...but it will be a moot point when we start using cqm-parsers to import the measures directly from the published MAT artifacts instead of our json format.